drivers/bridge/bridge.go

```diff
@@ -218,14 +225,24 @@ func (c *networkConfiguration) fromLabels(labels map[string]string) error {
 			if c.EnableICC, err = strconv.ParseBool(value); err != nil {
 				return parseErr(label, value, err.Error())
 			}
-		case DefaultBridge:
-			if c.DefaultBridge, err = strconv.ParseBool(value); err != nil {
-				return parseErr(label, value, err.Error())
-			}
 		case DefaultBindingIP:
 			if c.DefaultBindingIP = net.ParseIP(value); c.DefaultBindingIP == nil {
 				return parseErr(label, value, "nil ip")
 			}
+		case DefaultBridgeLegacyIPv6Network:
```

Comment:
What I am suggesting is create the pool in IPAM from the caller of libnetwork(docker), have some way of not allocating the IP address for IPv6(in fact we can do it for all networks if mask <= 80) and then allocate the IP address during `SetIPAddress` when driver calls. WDYT?